README.md

```diff
@@ -28,6 +28,8 @@ Please file issues for other unsupported features.
 - `refineType(type, format)` - an (optional) function to call to apply to type based on the type and format of the JSON schema.
 - `extensions` - an array of extensions to pass [joi.extend](https://github.com/hapijs/joi/blob/master/API.md#extendextension).
 - `strictMode` - make schemas `strict(value)` with a default value of `false`.
+- `warnOnEmpty` - outputs a warning message on console if a schema is empty or considered empty - https://json-schema.org/latest/json-schema-core.html#rfc.section.4.3.1
+- `extendValidation` - gives the ability to extend the default joi validation schema produced by `enjoi`. Works for the root schema and for references.
```

Comment:
Concerned this sounds a lot like joi extensions.

Reply:
It sound similar, but the key is in the details. Maybe it can be called `extendJoiSchemas` with a bit addition to the description:
`extendValidation` - gives the ability to extend the default Joi validation schema produced by `enjoi`. Works for the root schema and for referenced ones. In contrast to extensions which extends on a per type level, this options extends on a per schema level.